include/zephyr/dt-bindings/gpio/gpio.h

```diff
@@ -81,6 +81,10 @@
 
 /* Note: Bits 15 downto 8 are reserved for SoC specific flags. */
 
+/** Configures GPIO interrupt to wakeup the system from low power mode.
+ */
+#define GPIO_INT_WAKEUP         (1u << 28)
```

Comment:
In some systems wakeup pins are not waking up the system through interrupts. This is the case on STM32, and from what I could check this is aslo the case on Atmel SAM... There is no IRQ line associated to it and not interrupt controller.

Can we think about an alternate naming or removing reference to interrupts ?